src/App.tsx

```diff
@@ -30,7 +32,8 @@ const pageContainer = classnames(
 const bodyContainer = classnames(
   display('flex'),
   flexDirection('flex-col'),
-  margin('my-7', 'xxs:mx-auto', 'tablet:mt-0', 'tablet:mb-auto')
+  margin('my-7', 'xxs:mx-auto', 'tablet:mt-0', 'tablet:mb-auto'),
+  overflow('overflow-x-clip', 'sm:overflow-x-visible')
```

Comment:
Actually, we can clip it always, as I remember, nothing would break

Reply:
I checked it and didn't notice that smt is broken, but I remember that we had troubles with `overflow`, so I decided to return to default behavior to avoid any issues I didn't notice.